umap/static/umap/js/umap.features.js

```diff
@@ -271,6 +271,18 @@ L.U.FeatureMixin = {
     return value
   },
 
+  getSmartOption: function (option, fallback) {
```

Comment:
Not really fond of the `Smart` name, maybe `Dynamic`?

Reply:
Yep, smart is not smart choice :p

I've made the change :)